react-stage/src/services/CVService.js

```diff
@@ -36,7 +43,40 @@ const CVService = {
         const res = await fetch(urlBase + '/' + id)
         const data = await res.json()
         return data
-    }
+    },
+
+    getCvEtudiant: async (id) => {
+        const res = await fetch(urlBase + '/etudiant/' + id)
+        const data = await res.json()
+        return data;
+    },
+
+    deleteCv: async (id) => {
+        const res = await fetch(urlBase + '/delete/' + id, { method: 'DELETE' });
+        return res.ok;
+    },
+
+    saveCv: async (cv) => {
+        const res = await fetch(urlBase,
+            {
+                method: 'POST',
+                headers: {
+                    'Content-type': 'application/json'
+                },
+                body: JSON.stringify(cv)
+            })
+        if (res.ok) {
+            Swal.fire({
+                icon: 'success',
+                title: 'Succès!',
+                text: 'Votre cv vient d`être ajouté à la liste ci-dessous.'
+            })
+        }
+        const data = await res.json()
+        return data
```

Comment:
This function doesn't have proper error checking. It fires off an alert if the request was successful, but if the request failed, it still returns data and does not show an error alert